supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -61,7 +61,7 @@
 [discrete]
 [[active-directory-forest]]
 ==== image:images/yes.png[yes] Active Directory forest (noun)
-*Description*: An Active Directory (AD) forest is a set of one or more domain trees which share a common global catalog, directory schema, logical structure, and directory configuration. The forest represents the security boundary within which users, computers, groups, and other objects are accessible.
+*Description*: An _Active Directory_ (AD) forest is a set of one or more domain trees which share a common global catalog, directory schema, logical structure, and directory configuration. The forest represents the security boundary within which users, computers, groups, and other objects are accessible.
```

Comment:
I think you'd need to include "forest" in the italics here

Reply:
Updated.